### DISCUSSION TITLE
Talos 1.5.1 controlplane endpoint DNS query flood

### DISCUSSION THREAD

**Discussion by ruifung** [siderolabs/talos#7689] [Q&A]
Is it just me, or does it seem like talos 1.5.1 with kubeprism enabled seem to constantly do DNS queries for the controlplane endpoint?

I've noticed that since I've updated to 1.5.1, the top domain queried is my controlplane endpoint DNS at over 40k queries over the last hour from 6 nodes (3 control, 3 worker).

I just noticed this when looking at the stats on my local DNS server (Technitium DNS)

**Comment by smira** [comment]
See https://github.com/siderolabs/talos/issues/7690#issuecomment-1700783654
